app/controllers/user_controller.rb

```diff
@@ -157,9 +153,27 @@ def authorized?
     end
   end
 
-
   def user_params
-    params.require(:user).permit(:picture, :real_name, :orcid, :slug, :website, :location, :about, :preferred_locale, :help, :footer_block, notifications: [:user_activity, :owner_stats, :add_as_collaborator, :add_as_owner, :note_added, :add_as_reviewer])
+    params.require(:user).permit(
+      :picture,
+      :real_name,
+      :orcid,
+      :slug,
+      :website,
+      :location,
+      :about,
+      :preferred_locale,
+      :help,
+      :footer_block,
+      notification_attributes: [
```

Comment:
We can pass in `accepts_nested_attributes` for notification at `user` model so we can update them together.

Further optimization needed in update_profile logic.